error_reporting/google/cloud/error_reporting/client.py

```diff
@@ -12,13 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for interacting with the Stackdriver Logging API"""
+"""Client for interacting with the Stackdriver Error Reporting API"""
 
+import os
 import traceback
 
-import google.cloud.logging.client
+try:
```

Comment:
Having trouble finding a good example of an environment besides GAE standard where we have this problem. On GAE standard I get the import error `_args_from_interpreter_flags ` but I think that has to do general packaging weirdness on standard. Not sure best way to get development environment besides standard that I can't install gRPC on. Maybe we can get in touch with original author?